pkg/system/phase4_configuring.go

```diff
@@ -677,6 +689,106 @@ func (r *Reconciler) prepareGCPBackingStore() error {
 	return nil
 }
 
+func (r *Reconciler) prepareIBMBackingStore() error {
+	r.Logger.Info("Preparing backing store in IBM Cloud")
+
+	var (
+		endpoint string
+		location string
+	)
+
+	util.KubeCheck(r.IBMCloudCOSCreds)
```

Comment:
You can just get the secret using (example):
`
			secret := &corev1.Secret{
				TypeMeta: metav1.TypeMeta{Kind: "Secret"},
				ObjectMeta: metav1.ObjectMeta{
					Name:      Name,
					Namespace: Namespace,
				},
			}
if !util.KubeCheck(secret)
`
That way you will avoid adding secrets to the reconcile that you do not update or store anything in them.

Reply:
You are right, but I woudn't do that.
I think the current model is that all backing stores that have credentials are loaded into the reconciler struct for ease of use between the different phases.
So I would keep this working the same as the AWS and other credentials work.

Reply:
The thing is that in other credentials we are actually using these secrets in order to pull the credentials from the operator and assign them to our secret that won't get reconciled (with the addition of our information).
In this case, we do not do anything with the secret that we've created in the reconciler, since we just look and REF to the secret that we've listed.
So this is why I've said that we can remove this secret, but if you are saying that you prefer not to do the Kube check as I've suggested above, then we can leave this.